compat/poll/poll.c

```diff
@@ -147,7 +147,7 @@ win32_compute_revents (HANDLE h, int *p_sought)
       if (!once_only)
 	{
 	  NtQueryInformationFile = (PNtQueryInformationFile)
-	    GetProcAddress (GetModuleHandle ("ntdll.dll"),
+	    GetProcAddress (GetModuleHandleW (L"ntdll.dll"),
```

Comment:
Is there any reason to use the wide char version of the API over the narrow char here? It seems the pattern is to use wide char when using string literals with Win32 APIs, but to use the narrow char versions when using internal data from git... would it be simpler to just use always use narrow characters?